package.json

```diff
@@ -91,7 +91,7 @@
     "react-dom": "^16.2.0",
     "react-test-renderer": "^16.0.0",
     "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.3.0-artsy/relay-compiler-1.3.0-artsy.1.tgz",
-    "styled-components": "^2.2.3",
+    "styled-components": "^3.1.5",
```

Comment:
This looks great and excited about latest styles-components features :) One issue: Could you copy this line to 'peerDependencies' and then open up a corresponding 3.1.5 pr update in force?